packages/cli/src/cli/config/action.import.ts

```diff
@@ -307,14 +317,25 @@ export class CommandImport extends CommandLineAction {
   }
 
   _jobs: Map<string, CogStacJob> = new Map<string, CogStacJob>();
-  async _loadJob(path: string): Promise<CogStacJob> {
+  async _loadJob(path: string): Promise<CogStacJob | undefined> {
     const existing = this._jobs.get(path);
     if (existing) return existing;
+    const exist = await fsa.exists(path);
```

Comment:
this does two requests, and in theory the time between the exists and read the state could change (eg file deleleted or moved).

As a general rule you should attempt to read the file and look for a `404` if a error happens.

Reply:
Good point, I have remove the check exist and move read into try. Return null if failed to get the job.json.